schemas/webledger-validator.js

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright (c) 2017-2018 Digital Bazaar, Inc. All rights reserved.
+ */
+
+const createOperation = {
+  title: 'CreateWebLedgerRecord',
+  required: true,
+  type: 'object',
+  properties: {
+    type: {
+      type: 'string',
+      enum: ['CreateWebLedgerRecord'],
+      required: true
+    },
+    record: {
+      type: 'object',
+      required: true,
+      properties: {
+        id: {
+          type: 'string',
```

Comment:
should `record.id` be validated as a string or a URL?  Same goes for `recordPatch.target`.